graphics/vpype/Portfile

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                vpype
+version             1.9.0
+revision            0
+
+checksums           rmd160  b56bed854d574f69bf3c264d5c08c7061e238f6d \
+                    sha256  1b09fcd67ba911d8f558a09e3da6ae655ed921b7de599ebcacc7090550f77a71 \
+                    size    476328
+
+description         The Swiss-Army-knife command-line tool for plotter vector graphics.
+long_description    ${description}
+
+categories          graphics
+supported_archs     noarch
+maintainers         @abey79 openmaintainer
+license             MIT
+
+homepage            https://github.com/abey79/vpype
+
+python.default_version 39
+
+depends_build-append \
+                    port:py${python.version}-poetry-core
+
+depends_lib-append  port:py${python.version}-cachetools \
+                    port:py${python.version}-click \
+                    port:py${python.version}-multiprocess \
+                    port:py${python.version}-numpy \
+                    port:py${python.version}-pnoise \
+                    port:py${python.version}-shapely \
+                    port:py${python.version}-scipy \
+                    port:py${python.version}-svgelements \
+                    port:py${python.version}-svgwrite \
+                    port:py${python.version}-tomli \
+                    port:py${python.version}-asteval \
+                    port:py${python.version}-matplotlib \
+                    port:py${python.version}-glcontext \
+                    port:py${python.version}-moderngl \
+                    port:py${python.version}-pillow \
```

Comment:
this should be `Pillow` with a capital to make sure it will work properly on case-sensitive filesystems